### PR TITLE
Add Eventhub perf pipeline

### PR DESCRIPTION
**Pull request by g2vinay**
Adds required configuration to automate eventhubs perf.

### CODE REVIEW

**Review by mikeharder — APPROVED** [review]
LGTM, but we should create and run the pipeline to make sure it works before merging.

File: sdk/eventhubs/azure-messaging-eventhubs/perf-tests.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  'io.projectreactor:reactor-core': 3.4.27
+- 'com.azure:azure-messaging-eventhubs': source
+  'com.azure:azure-core': source
+  'com.azure:azure-core-amqp': source
+  'com.azure:azure-core-http-netty': source
+  'com.azure:azure-core-http-okhttp': source
+  'io.projectreactor:reactor-core': source
+
+Tests:
+- Test: send-event
```
Comment (pvaneck) [inline sdk/eventhubs/azure-messaging-eventhubs/perf-tests.yml:22]: Maybe make this test name more descriptive and consistent with Python: `send-event-batch`?

File: sdk/eventhubs/azure-messaging-eventhubs/perf-tests.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  Class: sendeventbatch
+  Arguments:
+  - --messagesize 1024 --events 100 --parallel 64
+
+- Test: receiive-eventprocessor
```
Comment (pvaneck) [inline sdk/eventhubs/azure-messaging-eventhubs/perf-tests.yml:27]: Likewise, perhaps `process-events-batch`?

File: sdk/eventhubs/azure-messaging-eventhubs/perf.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  type: string
+  default: '17'
+- name: PackageVersions
+  displayName: PackageVersions (regex of package versions to run)
+  type: string
+  default: '15|source'
+- name: Tests
+  displayName: Tests (regex of tests to run)
+  type: string
+  default: '^(send-events|event-processor-receive)$'
```
Comment (pvaneck) [inline sdk/eventhubs/azure-messaging-eventhubs/perf.yml:13]: This should match the test names in perf-tests.yml, right?

**Review by pvaneck — COMMENTED** [review]
Left some comments inline :)

**Pull request closed without merging**